front/src/components/Write/Stats.jsx

```diff
@@ -16,10 +18,10 @@ export default function WriteStats ({ stats }) {
       </h1>
       {expand && (
         <>
-          <p>Words : {stats.wordCount}</p>
-          <p>Characters : {stats.charCountNoSpace}</p>
-          <p>Characters (with spaces) : {stats.charCountPlusSpace}</p>
-          <p>Citations : {stats.citationNb}</p>
+          <p>{t('write.wordCount.stat', {stats: stats.wordCount})}</p>
```

Comment:
Le dernier élément de la clé devrait être "text"

```suggestion
          <p>{t('write.wordCountStat.text', {stats: stats.wordCount})}</p>
```